setup.py

```diff
@@ -7,12 +7,10 @@
 from codecs import open
 from os import path
 
-VERSION = str(shivyc.__version__)
-DOWNLOAD_URL = ("https://github.com/ShivamSarodia/ShivyC/archive/{}.tar.gz"
-                .format(VERSION))
+f"ShivyC only supports Python 3.6 or later"
 
-if sys.version_info[0] < 3:
-    sys.exit("ShivyC only supports Python 3.")
+VERSION = str(shivyc.__version__)
+DOWNLOAD_URL = f"https://github.com/ShivamSarodia/ShivyC/archive/{VERSION}.tar.gz"
```

Comment:
Curious - why remove the Python version check here?

Reply:
On any Python version that does not support f-strings the result of the new line above will be:
```
>>> +f"ShivyC only supports Python 3.6 or later"
  File "<string>", line 1
    +f"ShivyC only supports Python 3.6 or later"
                                               ^
SyntaxError: invalid syntax
```

Reply:
Ah, I see! Thanks.